packages/ember-data/lib/adapters/build-url-mixin.js

```diff
@@ -27,7 +27,7 @@ var get = Ember.get;
   @class BuildURLMixin
   @namespace DS
 */
-var BuildURLMixin = Ember.Mixin.create({
+export default Ember.Mixin.create({
```

Comment:
`<ctrl-p> buildurlm` :stuck_out_tongue: nicely targets the file that it comes from, not confused with consumers